lib/iex/lib/iex/helpers.ex

```diff
@@ -643,4 +644,42 @@ defmodule IEx.Helpers do
              Integer.to_char_list(z) ++ '>'
     )
   end
+
+  @doc """
+  Deloys a given module's beam code to a list of connected nodes.
+
+  This function is useful for development and debugging when you have code that
+  has been compiled or updated locally that you want to run on other nodes.
+  
+  The node list defaults to a list of all connected nodes.
+
+  Returns `{:error, :nofile}` if the object code (i.e. ".beam" file) for the module
+  could not be found locally.
+
+   ## Examples
+
+      nl(HelloWorld)
+      #=> {:ok, [{:node1@easthost, :loaded, HelloWorld},
+                 {:node1@westhost, :loaded, HelloWorld}]}
+
+      iex> nl(NoSuchModuleExists)
+      {:error, :nofile}
```

Comment:
I think you used `iex>` just on one of the two examples because only the second one can be easily doctested; I think you shouldn't use `iex>` in the second example either, for consistency. Let's hear what other people think :)